doc/api/net.md

```diff
@@ -902,6 +905,19 @@ For TCP connections, available `options` are:
   **Default:** `false`.
 * `keepAliveInitialDelay` {number} If set to a positive number, it sets the initial delay before
   the first keepalive probe is sent on an idle socket.**Default:** `0`.
+* `autoSelectFamily` {boolean|number}: If set to `true` or to a positive number, it enables
```

Comment:
It feels strange to mix `boolean|number` here instead of splitting it into two options. The meaning of the `number` is also not obvious and the reader must read the entire paragraph to understand its meaning.

Reply:
You are right. It's confusing. I'll split in `autoSelectFamily` and `autoSelectFamilyAttemptTimeout`.